config/routes.yaml

```diff
@@ -9,6 +9,13 @@ chat:
       path: 'https://discord.gg/PruNxpG'
       permanent: true
 
+bolresults:
```

Comment:
If we have future BoL seasons, what would we do? Update the redirect URL?

I would recommend use a separate path for each BoL entries

Reply:
I thought about that, but that would end up with an impressively long routes file as there are currently nine volumes, with no plans of stopping in the future. Additionally, I intend to make a later (like, in a year haha) PR that adds a short link for submission. It is mostly for convenience and branding, but for now making a PR to update to the most recent results once a year seems okay to me.

Reply:
@liushuyu is right, routes for each result.  Length isn't a problem.